piptools/scripts/compile.py

```diff
@@ -383,8 +390,8 @@ def cli(
                 from distutils.core import run_setup
 
                 dist = run_setup(src_file)
-                tmpfile.write("\n".join(dist.install_requires))
-                comes_from = f"{dist.get_name()} ({src_file})"
+                tmpfile.write("\n".join(dist.install_requires))  # type: ignore
```

Comment:
Instead of `# type: ignore`, it's better to use `# type: ignore[error-code1,error-code2]` etc

Reply:
Yeah, that's a good point. It's WIP and I thought I could find some appropriate type to cast.

Reply:
Thank. Addressed in 7faaded935908b74e22483554e008c9486a03bb6.